searchcore/src/vespa/searchcore/proton/server/documentdb.cpp

```diff
@@ -805,9 +807,9 @@ DocumentDB::setIndexSchema(const DocumentDBConfig &configSnapshot, SerialNum ser
 }
 
 void
-DocumentDB::reconfigure(const DocumentDBConfig::SP & snapshot)
+DocumentDB::reconfigure(DocumentDBConfig::SP snapshot)
 {
-    masterExecute([this, snapshot]() { newConfigSnapshot(snapshot); });
+    masterExecute([this, snapshot=std::move(snapshot)]() mutable { newConfigSnapshot(std::move(snapshot)); });
```

Comment:
Consider removing the first std::move() of snapshot on this line, snapshot is needed a few lines below.